src/raftstore/store/peer.rs

```diff
@@ -246,6 +246,10 @@ pub struct Peer {
     //      locally.
     leader_lease_expired_time: Option<Either<Timespec, Timespec>>,
 
+    // If a snapshot is being applied asynchronously, messages should not be sent untill
```

Comment:
do we need to use `until` here? @queenypingcap

Reply:
Yes. But please use "until" instead of "untill".